src/query-helpers.js

```diff
@@ -32,6 +32,7 @@ function getElementError(message, container) {
 function firstResultOrNull(queryFunction, ...args) {
```

Comment:
should this be renamed to something like "onlyResultOrNull" since it is more strict that the length has to be 1 now?

Reply:
Whoops, I didn't mean to change that yet 😬